src/Monolog/Formatter/ElasticaFormatter.php

```diff
@@ -79,10 +79,14 @@ public function getType()
      */
     protected function getDocument($record)
     {
+        // index should be timestamped to allow log rotating
+        $date = str_replace('-', '.', explode('T', $record['datetime'])[0]);
```

Comment:
`$date = $record['datetime']->format('Y.m.d')` would achieve the same in a more obvious way as far as I can tell.